lib/effects/scale_effect.dart

```diff
@@ -1,23 +1,22 @@
 import 'package:flutter/animation.dart';
 import 'package:meta/meta.dart';
 
-import 'dart:ui';
 import 'dart:math';
 
+import '../vector2_extension.dart';
 import './effects.dart';
-import '../position.dart';
 
 double _direction(double p, double d) => (p - d).sign;
 double _size(double a, double b) => (a - b).abs();
```

Comment:
I think calling this `dist` or `mod` will be a bit more clear than `size` witch now seems to imply a Vector2

Reply:
Settled for `length`